pkg/snet/network.go

```diff
@@ -21,9 +21,9 @@ import (
 // Default ports.
 // TODO(evanlinjin): Define these properly. These are currently random.
 const (
-	SetupPort      = uint16(36)  // Listening port of a setup node.
-	AwaitSetupPort = uint16(136) // Listening port of a visor node for setup operations.
-	TransportPort  = uint16(45)  // Listening port of a visor node for incoming transports.
+	SetupPort      = uint16(36)  // Listening port of a setup visor.
```

Comment:
`setup node` (this is the last such comment, please check if `setup visor` exists below)